src/utils/recordEvent.js

```diff
@@ -19,6 +19,7 @@ export const recordEvent = (data) => {
  * @param {string} key the property in the event object to use when looking for
  *                     existing matches in the dataLayer
  */
+/* eslint-disable-next-line typescript-eslint/no-shadow */
```

Comment:
storybook was having trouble compiling the news_story component without this piece... i think some rule under the hood from their eslint extension and the fact this is a .js module being imported. it doesn't like the `(event,` because we import `event` at the top of the file.

Reply:
yeah, that's the analytics beacon causes a bunch of issues in tests also